x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_list_filter.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { memo, useCallback } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSelectable, EuiPopoverTitle } from '@elastic/eui';
+import { ActionListFilterPopover } from './action_list_filter_popover';
+import { type FilterItems, type FilterName, useActionListFilter } from './hooks';
+import { ClearAllButton } from './clear_all_button';
+
+const getFilterName = (name: string) =>
+  i18n.translate('xpack.securitySolution.responseActionsList.list.filterName', {
+    defaultMessage: `{name}`,
+    values: { name },
+  });
```

Comment:
Ok. You need to explain this one 😆 
So - the get file name just return the file name that was passed in? does not make sense. There is nothing to localize

Reply:
Oh! I intended to translate the filter name here that shows up on the filpter popup. Will fix this.

Reply:
removed this bit 22e0c598c6bdfbb92de18cd6cc7d427c8dae32f9